### PR TITLE
exit(0) on --version

### PR DESCRIPTION
**Pull request by lifeofguenter**
`--version` on commands should not exit with an error.

### CODE REVIEW

File: script/munin-html [hunk 1]
```diff
@@ -80,6 +80,7 @@ sub print_version_and_exit {
     Pod::Usage::pod2usage(
         -verbose  => 99,
         -sections => 'VERSION|COPYRIGHT',
+		-exit     => 0,
```
Comment (kenyon) [inline script/munin-html:83]:
```suggestion
        -exit     => 0,
```

File: script/munin-graph [hunk 2]
```diff
@@ -85,6 +85,7 @@ sub print_version_and_exit {
     Pod::Usage::pod2usage(
         -verbose  => 99,
         -sections => 'VERSION|COPYRIGHT',
+		-exit     => 0,
```
Comment (kenyon) [inline script/munin-graph:88]:
```suggestion
        -exit     => 0,
```

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,15 @@
+root = true
```
Comment (kenyon) [inline .editorconfig:1]: Unrelated change, should be removed from this commit.

File: .gitignore [hunk 4]
```diff
@@ -29,6 +29,7 @@ local/
 # editor files
 .idea/
 .*.sw[op]
+/*.iml
```
Comment (kenyon) [inline .gitignore:32]: Unrelated change, should be removed from this commit.

**Review by kenyon — CHANGES_REQUESTED** [review]

File: .editorconfig (hunk 3 above)
Comment (lifeofguenter) [reply .editorconfig:1]: not unrelated, but something I would like to introduce as well :)

File: .gitignore (hunk 4 above)
Comment (lifeofguenter) [reply .gitignore:32]: I see other editor files being ignored as well, why not?

File: .editorconfig (hunk 3 above)
Comment (kenyon) [reply .editorconfig:1]: Introduce in a separate pull request then.

File: .gitignore (hunk 4 above)
Comment (kenyon) [reply .gitignore:32]: Should be a separate pull request.
